brands/tourism/hotel.json

```diff
@@ -121,6 +121,17 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Clarion": {
+    "countryCodes": ["us"],
+    "nomatch": ["tourism/hotel|Comfort Suites", "tourism/hotel|Comfort Inn"],
```

Comment:
These shouldn't be necessary here

Reply:
HI ! I'm sorry, do you mean the whole pull request is necessary, or lines 124 through 126?

Reply:
@tas50 means the above line is unnecessary and can be removed.
Edit your commit to remove it.

Reply:
@bmillemathias  thank you !